layers: [200, 10, 1]
alpha: 1
beta: 1
training_algorithm: BackPropagation
learning_rate: 0.0001
momentum_scale: 0.1
